README.md

```diff
@@ -22,3 +22,111 @@ features made available by `ocaml-lsp-server`. Its aim is to offer a
 user experience as close as possible to that offered by the Emacs mode
 [Merlin](https://ocaml.github.io/merlin/editor/emacs/).
 
+## Features
+
+### Browsing errors
+
+Eglot relies on
+[Flymake](https://www.gnu.org/software/emacs/manual/html_node/emacs/Flymake.html)
+for error diagnosis. OCaml-eglot offers two functions for quickly
+navigating through errors:
+
+- `ocaml-eglot-error-next` (<kbd>C-c</kbd> <kbd>C-x</kbd>): jump to
+  the next error
+- `ocaml-eglot-error-prev` (<kbd>C-c</kbd> <kbd>C-c</kbd>): jump to
+  the previous error
+
+![Error navigation example](media/error-navigation.gif)
+
+### Jump to definition
+
+Eglot relies on
+[Xref](https://www.gnu.org/software/emacs/manual/html_node/emacs/Xref.html)
+to index cross-references. OCaml-eglot provides a shortcut to quickly
+jump to the definition:
+
+- `ocaml-eglot-locate` (<kbd>C-c</kbd> <kbd>C-l</kbd>): jump to
+  definition
+
+![Jump to definition example](media/locate.gif)
+
+### Infer Interface
+
+Used to infer the type of an interface file. If the buffer is not
+empty, a prompt will ask for confirmation to overwrite the buffer
+contents:
```

Comment:
Reading only the text, it's unclear of which "non-empty buffer" your talking about.

I guess the command should be called from within an implementation file and will create a new buffer for the interface. It is that new buffer that might erase an existing one right ? I think it could be made a bit clearer.

Reply:
And now that I looked at the gif, I understand it is the other way around :-)

For me it makes more sense to (also) be able to call infer-interface from the implementation itself but I don't have a strong opinion.